modules/rgbd/perf/perf_tsdf.cpp

```diff
@@ -169,51 +169,90 @@ Ptr<Scene> Scene::create(Size sz, Matx33f _intr, float _depthFactor)
     return makePtr<SemisphereScene>(sz, _intr, _depthFactor);
 }
 
-PERF_TEST(Perf_TSDF, integrate)
+struct Settings
 {
     Ptr<kinfu::Params> _params;
-    _params = kinfu::Params::coarseParams();
+    Ptr<kinfu::Volume> volume;
+    Ptr<Scene> scene;
+    std::vector<Affine3f> poses;
+};
+
+Settings setSettings(bool useHashTSDF)
```

Comment:
Can be implemented as a constructor for `Settings` class

Reply:
done